src/scripts/main.js

```diff
@@ -2,6 +2,40 @@
 
 const pushNotification = (posTop, posRight, title, description, type) => {
   // write code here
+  const parentElem = document.body;
+  const newDiv = document.createElement('div');
+  const titleElem = document.createElement('h2');
+  const descriptionELem = document.createElement('p');
+
+  newDiv.className = 'notification';
+  titleElem.className = 'title';
+  descriptionELem.className = 'description';
+  descriptionELem.textContent = description;
+  descriptionELem.style.whiteSpace = 'pre-wrap';
+  titleElem.textContent = title;
+
+  parentElem.prepend(newDiv);
+  newDiv.append(titleElem);
+  newDiv.append(descriptionELem);
+
+  if (type === 'success') {
+    newDiv.classList.add('success');
+  }
+
+  if (type === 'error') {
+    newDiv.classList.add('error');
+  }
+
+  if (type === 'warning') {
+    newDiv.classList.add('warning');
+  }
```

Comment:
Currently, the function only handles 'success', 'error', and 'warning' types. Consider adding a default case or handling for other notification types to make the function more robust.